scripts/prepare_docker_buildinfo.sh

```diff
@@ -11,10 +11,20 @@ DISTRO=$5
 DOCKERFILE_PATH=$(dirname "$DOCKERFILE_TARGE")
 BUILDINFO_PATH="${DOCKERFILE_PATH}/buildinfo"
 BUILDINFO_VERSION_PATH="${BUILDINFO_PATH}/versions"
+if [[ $CROSS_BUILD_ENVIRON == y ]]; then
```

Comment:
>$CROSS_BUILD_ENVIRON [](http://example.com/codeflow?start=6&length=20)

This file is to implement reproducible feature [https://github.com/Azure/SONiC/pull/684](https://github.com/Azure/SONiC/pull/684). Why you add here?

Reply:
I need to copy prebuilt (cross-compiled) python wheels to the dockers directory in order to be installed in the dockers Docker.j2 file. See for example dockers/docker-config-engine-buster/Dockerfile.j2.
Please suggest what and where to do it if this file is inappropriate.
Thanks.